src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+  const propertyValues = Object.values(robot);
+  let current;
```

Comment:
The variable 'current' is declared too early. It is not used until line 15, so it would be better to declare it there to improve readability.